main.py

```diff
@@ -1,11 +1,59 @@
+from typing import Any
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
+def get_or_create_race(player: dict) -> Any:
+    race_inst, created = Race.objects.get_or_create(
+        name=player["race"]["name"],
+        defaults={"description": player["race"]["description"]}
+    )
+    return race_inst
+
+
+def get_or_create_skill(player: dict) -> None:
+    for skill in player["race"]["skills"]:
+        Skill.objects.get_or_create(
+            name=skill["name"],
+            defaults={
+                "bonus": skill["bonus"],
+                "race": get_or_create_race(player)
```

Comment:
move it before the loop, your code is unoptimized, because in each step you do two queries instead of one in the loop and one before the loop